about.html

```diff
@@ -0,0 +1,166 @@
+                    </li>
+                    <li>
+                        <a class="menu-item" href="#">Sponsor</a>
+                    </li>
+                    <li>
+                        <a class="menu-item" href="#">News</a>
+                    </li>
+                    <li>
+                        <a class="menu-item" href="#">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+
+            <div class="hero1">
+                <h1 class="heading">
+                    "Hello, Welcome !!
+                </h1>
+                <h2 class="sec_head1">
+                    To Web Developer Conference 2022
+                </h2>
+                <div class="about_d">
+                    <p class="first_para1">
+                        We will inform you without obligation about all news about the WDC - Web Developer Conference.
+                        You will receive exclusive offers free of charge.
+                    </p>
+                </div>
+                <div class="about_contact">
+                    <p class="contact_info">
+                        Please contact us per Email for any further enquiries
+                        about the Web Developer Conference 2022!
+                    </p>
+                    <a class="about_email" href="abenagy16@gmail.com">abenagy16@gmail.com</a>
+                </div>
+            </div>
+        </header>
+    </section>
+
+    <section class="logo_sect">
+        <div class="main_about">
+            <h3 class="pro_head">Web Developer Conference 2022 logo</h3>
+            <hr class="about_hr">
+        </div>
+        <p class="logo_description">
+            The Logo of the Web Developer Conference 2022 was descided through the
+            logo competition from 8. june to 11. july
+        </p>
+        <div class="conference_img">
+            <img class="logo" src="./images/nav.png" alt="Web Developer Conference logo">
+        </div>
+    </section>
+
+    <section class="history">
+        <div class="main_history">
+            <h3 class="head_history">See the History</h3>
+            <hr class="history_hr">
+        </div>
+        <p class="history_description">
+            Have a look at the two last Web Developer Conference which took place at Empire Riverside Hotel
+        </p>
+        <div class="history_img-div ">
+            <div class="img_h overlay">
+                <img class="hist_img" src="./images/history_1.jpeg" alt="Web Developer Conference logo">
+                <p class="h_para"><span class="year_hist">2021</span>Web Developer Conference at Empire Riverside
+                    Hotel</p>
+                <img class="hist_img" src="./images/history_2.jpeg" alt="Web Developer Conference logo">
+                <p class="h_para"><span class="year_hist">2018</span>Web Developer Conference at Empire Riverside
+                    Hotel</p>
+            </div>
+        </div>
```

Comment:
- [ ] The text in the images on the mobile version is not perfectly centered and the last line is cut out of the image. Kindly fix the position of the text so it can show the entire text centered in the image.
 - [ ] On the desktop view the gap between the images does not exist, and the left image is smaller than the right one on some screen sizes. It is also not perfectly centered on the page on some screen sizes and the text in the images is not perfectly centered. Kindly fix those issues so your images can look like the template you follow.
 - Your mobile version:
 - 
<img width="373" alt="image" src="https://user-images.githubusercontent.com/61919617/204301661-0ebfcee0-fd79-4e73-bb06-0caa00c28cd3.png">

 - Your desktop version:
 - 
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/61919617/204301770-cb8b1049-94cf-403d-897b-af5bb73e6223.png">

 - Template mobile:
 - 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/61919617/204301843-54aab3a4-3ee2-477a-b935-9e0550eab5ac.png">

 - Template desktop:
 - 
<img width="644" alt="image" src="https://user-images.githubusercontent.com/61919617/204301937-9f2740f9-a5d9-453b-b1ba-fc423d75811d.png">

Reply:
-  Text in the images on the mobile version is perfectly centered .
-  On the desktop view the gap between the images now exist and the text perfectly centered.
-